airflow/providers/amazon/aws/operators/datasync.py

```diff
@@ -350,6 +350,16 @@ def _execute_datasync_task(self):
         )
         self.log.info("task_execution_description=%s",
                       task_execution_description)
+
+        # Log some meaningful statuses
+        log = self.log.info
+        if not result:
+            log = self.log.error
```

Comment:
This is a slightly odd way. Instead could you do:

```python
        level = logging.ERROR if not result else logging.INFO
        self.log.log(level, 'Status=%s', task_execution_description['Status'])
```

etc. (Probably need `import logging` at the top of the file to get access to those constants)

Reply:
Thanks, your way is better. I'll update :)